### PR TITLE
Update notebook readme

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+To install the library, run:
 
 ```bash
 pip install git+https://github.com/datacommonsorg/llm-tools
 ```
 
-## Examples
-
-TODO: Add links
+To see examples of using this library, can check out our Colab notebooks for [RIG](https://github.com/datacommonsorg/llm-tools/blob/main/notebooks/data_gemma_rig.ipynb)
```
Comment (chejennifer) [inline README.md]: maybe instead, "For examples of using this library, see our Colab notebooks for [RIG](...) and [RAG](...)"

**Review by chejennifer — APPROVED** [review]
nice

File: README.md (hunk 1 above)
Comment (shifucun) [reply README.md]: updated

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**